proto/cosmos/bank/v1beta1/authz.proto

```diff
@@ -17,3 +18,14 @@ message SendAuthorization {
   repeated cosmos.base.v1beta1.Coin spend_limit = 1
       [(gogoproto.nullable) = false, (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"];
 }
+
+// PeriodicSendAuthorization allows the grantee to spend up to periodic_allowance.period_spend_limit coins from
+// the granter's account during a period of periodic_allowance.period seconds
+//
+message PeriodicSendAuthorization {
+  option (cosmos_proto.implements_interface) = "Authorization";
+
+  // periodicAllowance specifies a struct of `PeriodicAllowance`
+  cosmos.feegrant.v1beta1.PeriodicAllowance periodic_allowance = 1 [(gogoproto.nullable) = false];
```

Comment:
We shouldn't use feegrant's PeriodicAllowance in authz. We can create a new PeriodicAllowance for authz only. This is to leave the possibility of both PeriodicAllowances to diverge, without proto-breaking changes.

Reply:
Yeah, I figured I would get that comment, but used this as a proof of concept implementation as it was the easiest way to test it.